src/metabase/db/liquibase.clj

```diff
@@ -204,7 +205,10 @@
 (defn force-release-locks!
   "(Attempt to) force release Liquibase migration locks."
   [^Liquibase liquibase]
-  (.forceReleaseLocks liquibase))
+  (if-not (= (mdb.connection/db-type) :h2)
+    (log/warnf "Database %s does not support forcing migration locks. The lock is always automatically removed when the process performing the migration completes or fails."
```

Comment:
Why not make this a failure?

Reply:
Thought about this a bit more, now I'm pretty sure that it should throw, and instead `release-lock-if-needed!` shouldn't even try if the conn is not h2.